src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator is stopped working");

```

Comment:
Similarly, the stop message 'Excavator is stopped working' should be more informative and consistent. A suggested rephrase could be 'Excavator has stopped its work'.